src/buttons/buttons.js

```diff
@@ -1,12 +1,12 @@
 angular.module('mm.foundation.buttons', [])
 
 .constant('buttonConfig', {
-    activeClass: 'active',
+    deactiveClass: 'hollow',
```

Comment:
Cool thanks. It'll also be easier for people moving from angular-foundation as they might have set their own 'activeClass' via the config. :)